tutorial/README.md

```diff
@@ -3,3 +3,35 @@
 Code for the [Bazel Haskell tutorial][bazel-haskell-tutorial].
 
 [bazel-haskell-tutorial]: https://rules-haskell.readthedocs.io/en/latest/haskell.html
+
+## **Important**
+
+Run all commands from the root of `rules_haskell`.
+If you `cd examples/`, bazel *will* [break on
```

Comment:
"if you `cd tutorial/`".

Reply:
Does this mean that the instructions in the tutorial itself are broken? If so that's a big deal, and we'll need to find a better workaround.

Reply:
Hm, it looks like it does work for `tutorial/` (not for `examples/` though).